### ISSUE TITLE
Add support for managing docker containers

### ISSUE THREAD

**Issue by lilasquared** [jmal98/ansiblecm#8]
Would it be possible to add the dependencies listed on [this page](https://docs.ansible.com/ansible/latest/modules/docker_container_module.html) so that this image could control docker containers?  Or would it make more sense to create an image using this as the base that has these dependencies?

**Comment by jmal98** [comment]
Yup, adding the dependencies to this image would be fine.  Would you be able to make the changes and validate them?

**Comment by lilasquared** [comment]
I tried adding the docker package to the pip install step, however i am getting an error

"Failed to import docker or docker-py (Docker SDK for Python) - No module named requests.exceptions. Try `pip install docker` or `pip install docker-py` (Python 2.6)."

I assume this package is using python 2.7 from this line `python2-dev=2.7.16-r1 \`

Any thoughts on what I might be doing wrong?

**Comment by lilasquared** [comment]
@jmal98 I think I read the documentation wrong, it looks like the dependencies are required on the **remote host** not the control machine.  If I get it working with no change to the current image version I will close this.

**Comment by jmal98** [comment]
ok
